### ISSUE TITLE
The tool tip is provided when we hover over "Azure App Services" in the list using mouse, but is not accessible using keyboard arrow keys.

### ISSUE THREAD

**Issue by v-gitsay** [microsoft/vscode#135639]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;#DesktopApp;#WCAG2.1.1;#AINotInScope;#FTP;#Keyboard;

### Environment Details:
Screen Reader: NVDA/JAWS
Operating system: Windows 10

### Repro Steps:

1. Launch VS Code application.
2. Tab till Extensions in the Navigation bar and press enter.
3. Tab to search box.
4. Type "Azure App Services" in search bar and Tab to the "Azure App Services" in the resulting list.
5. Check if the tool tip is accessible for "Azure App Services" in the list using keyboard .

### Actual:
The tool tip is provided when we hover over "Azure App Services" in the list using mouse, but is not accessible using keyboard arrow keys.

### Expected:
Tool tip should be accessible for "Azure App Services" in the list using keyboard arrow keys.

### User Impact:
Keyboard users will not get complete information about the list item "Azure App Services" .

**Please find the below attachment for reference:**
**![134296946-13d8dd37-c537-4d3b-8798-8cae497bd59b](https://user-images.githubusercontent.com/86710911/138442209-2ad292ed-351b-4657-9002-2b83afd37d73.gif)**
